src/components/OcSelect.vue

```diff
@@ -32,6 +35,11 @@ export default {
       required: false,
       default: null,
     },
+    labelText: {
```

Comment:
Would it make sense to bind `$attrs` without the `label` prop to the vue-select? So that we can have a dedicated `label` prop on the OcSelect? I kind of dislike using `labelText` as prop name here, while we use `label` on all other form related elements.

Reply:
Full ack! Streamline all the things 🥳